docs/application/dotnet/guides/applications/service_application.md

```diff
@@ -0,0 +1,239 @@
+# Service Applications 
+
+Service applications are Tizen .NET applications with no graphical user interface that run in the background. They can be very useful in performing activities (such as getting sensor data in the background) that need to run periodically or continuously, but do not require any user intervention.
+
+The service application type allows to create reusable and independent parts which is really important in bigger projects where we can easily split responsibilities of the application to different parts. As an example, consider the speedometer application designed for average speed measurement. Service part in this case is responsible for reading speed from the device GPS module. It also calculates average speed. The UI communicates with the service when it is visible and shows measured values. That approach allows to reuse module between different applications such as a cyclist application, automotive solutions, sport activity apps. Core module with business logic works in the background and carry out designed things until it gets closed by the system evens like `OnLowMemory` or `OnLowBattery` where UI app could be accidentally closed by the user.
+
+The main Service Application API features include: 
+-  Application States:
+
+    A Tizen native service application has different states which it transitions through during its life-cycle.
```

Comment:
This page seems to be copied from the page under native so there are several 'native' strings. Please incorporate the suggestion if you think this is correct or let me know otherwise.
```suggestion
    A Tizen service application has different states which it transitions through during its life-cycle.
```